src/main/java/com/moddy/server/external/kakao/dto/response/KakaoUserResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.moddy.server.external.kakao.dto.response;
+
+
+import lombok.*;
+
+@ToString
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+
+public class KakaoUserResponse {
+
+    private Long id;
+
+
+}
```

Comment:
```suggestion
@ToString
@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor(access = AccessLevel.PRIVATE)

public record KakaoUserResponse(Long id) {
}
```
p3
java 17 부터는 record를 사용해서 표현할 수 있습니다!
record를 통해 위 어노테이션을 전부 없앨 수 있습니다
참고해주세요